### PR TITLE
fix(curriculum): Missing punctuation in user stories

### LINKED ISSUE #56813: Missing punctuation in user stories
### Describe the Issue

This document has no full-stop at the end of each "User Stories".

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/build-a-palindrome-checker-project/build-a-palindrome-checker

### Your code

At the end of each "User Stories" add punctuation.

### Expected behavior

.

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by khushi144**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Added full stop.

Closes #56813

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-palindrome-checker-project/build-a-palindrome-checker.md
Comment (ilenia-magoni) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-palindrome-checker-project/build-a-palindrome-checker.md:1]: line 15 is also missing a period

Comment (khushi144) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-palindrome-checker-project/build-a-palindrome-checker.md:1]: @ilenia-magoni I have resolved this. Is my PR ready for merged now?

Comment (ilenia-magoni) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-palindrome-checker-project/build-a-palindrome-checker.md:1]: no, it needs to have two approving reviews

**Comment by gikf** [comment]
Could you make such change to other JavaScript projects as well?

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-cash-register-project/build-a-cash-register.md
https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-roman-numeral-converter-project/build-a-roman-numeral-converter.md
https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-telephone-number-validator-project/build-a-telephone-number-validator.md

- - -

Note for future - it is recommended to make changes on a separate branch, rather than the `main` branch. See more about it in point 3 at https://contribute.freecodecamp.org/how-to-contribute-to-the-codebase/#contributing-to-the-codebase.

**Comment by khushi144** [comment]
Should make such changes in this PR or I should create seperate issues?

> Note for future - it is recommended to make changes on a separate branch, rather than the `main` branch. See more about it in point 3 at https://contribute.freecodecamp.org/how-to-contribute-to-the-codebase/#contributing-to-the-codebase.

I will take care from next time.

**Comment by ilenia-magoni** [comment]
As it was asked in this PR you can do so in this PR

**Comment by gikf** [comment]
> For other JS projects should I make such changes in this PR or I should create seperate issues?

Since it is the exact same problem, and fixing in to other projects will not bloat the PR, you can make changes here here. Just push further commits to the same branch on GitHub, and new changes will be reflected in the PR.

**Review by gikf — APPROVED** [review]

**Review by Dario-DC — APPROVED** [review]

**Comment by Dario-DC** [comment]
Hi @khushi144 . Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Comment by khushi144** [comment]
Thank you :)

**Pull request merged**